helpers/exceptions.js

```diff
@@ -63,5 +63,6 @@ module.exports = {
 		'5524930565698900323',  // 20407
 		'11613486949732674475', // 123300
 		'14164134775432642506'  // 123333
-	]
+	],
+	blockHeightToDisableDappTransfers: 0
```

Comment:
Can we move this to a new file `helpers/milestones.js`? We previously had this, but was recently removed as unused.

```js
module.exports = {
    disableDappTransfers: 0
};
```

I say this because an exception serves a different purpose.